js/main.js

```diff
@@ -48,6 +51,38 @@ $brName.addEventListener('keydown', function (e) {
   }
 });
 
+const $brCityList = document.querySelector('#city-list');
+
+$brCity.addEventListener('keydown', e => {
+  if (event.code === 'Enter') {
+    event.preventDefault();
+
+    const $brList = document.createElement('ul');
+    $brList.className = 'br-list';
```

Comment:
Instead of creating the `ul` inside of here, why not create the `ul` in the HTML, and just append everything inside the `ul`?